packages/docsearch-react/package.json

```diff
@@ -9,7 +9,6 @@
     "name": "Algolia, Inc.",
     "url": "https://www.algolia.com"
   },
-  "sideEffects": false,
```

Comment:
it should be this line, not the one you changed, this is the package.json for the react package, not the css package

https://github.com/algolia/docsearch/blob/e68441b51cecf5170e9d01de37baa1bd25956378/packages/docsearch-css/package.json#L12

Reply:
But I was using `@docsearch/css` package in my case. There's no problem with `@docsearch/react` package even with `sideEffects: false` presented in its `package.json`. Do you want me to set up a repository so you can check?

Reply:
the change in this PR was made to the react package, instead of the css package, you might have confused the two?

Reply:
> the change in this PR was made to the react package, instead of the css package, you might have confused the two?

Oops, sorry, it's my fault.

Reply:
Updated, thanks for heads-up.